openghg_inversions/hbmcmc/inversion_pymc.py

```diff
@@ -285,7 +285,7 @@ def inferpymc(
             pollution_event = np.abs(pt.dot(hx, x))
 
         pollution_event_scaled_error = pollution_event * sig[sites, sigma_freq_index]
-        epsilon = pt.sqrt(error**2 + pollution_event_scaled_error**2 + min_error**2)
+        epsilon = pt.maximum(pt.sqrt(error**2 + pollution_event_scaled_error**2), min_error)
```

Comment:
```suggestion
        epsilon = pt.max(pt.sqrt(error**2 + pollution_event_scaled_error**2), min_error)
```

Reply:
Hmmm I think pt.maximum and pt.max do different things (https://pytensor.readthedocs.io/en/latest/library/tensor/basic.html#pytensor.tensor.maximum). I'm not certain that pt.maximum is right though, so you better check!